docs/installation.mdx

```diff
@@ -123,4 +123,27 @@ Version:        1.0.0
 Go version:     go1.20.2
 Git commit:     b58e7b910ca556973d111e9bd734a71baef03db2
 Git tree state: clean
+```
+
+## To Validate the Checksum of ORAS CLI Binaries
+
+
+The following commands can be used to verify the ORAS CLI binaries using GPG:
+
+1. First, we import a GPG Key which can be used for verification:
+
+```
+$ curl -sSL https://github.com/qweeah.gpg | gpg --import -
+gpg: key 276D8A724CE1C704: "Billy Zha <jinzha1@microsoft.com>" not changed
+gpg: Total number processed: 1
+gpg: unchanged: 1
```

Comment:
```suggestion
```
Add  command like below to confirm the key is downloaded and imported
```console
$ gpg --list-keys
...
pub   rsa4096 2023-02-28 [SC] [expires: 2024-02-28]
      BE6FA8DDA48D4C230091A0A9276D8A724CE1C704
uid           [ultimate] Billy Zha <jinzha1@microsoft.com>
...
```

Reply:
Thank you!